### PR TITLE
Rename `threads` param to `num_threads`.

### PR DESCRIPTION
**Pull request by frewsxcv**
An attempt to differentiate/clarify the parameter name.

### CODE REVIEW

**Comment by dns2utf8** [comment]
I like the change, but it seems not yet complete.

**Comment by frewsxcv** [comment]
@dns2utf8 Want to review this?

File: src/lib.rs [hunk 1]
```diff
@@ -209,15 +209,25 @@ impl ThreadPool {
...
-        let current_max = (*self.max_count).swap(threads, Ordering::Release);
-        if threads > current_max {
+    /// **Deprecated: Use `ThreadPool::set_num_threads`**
+    // #[deprecated(since = "1.3.0", note = "use ThreadPool::set_num_threads")]
+    // TODO: #[deprecated] isn't stable yet.
+    pub fn set_threads(&mut self, num_threads: usize) {
+        self.set_num_threads(num_threads)
+    }
+
+    /// Sets the number of threads to use as `num_threads`.
```
Comment (dns2utf8) [inline src/lib.rs]: Maybe rephrase to "number of workers" or "number of working threads"

File: src/lib.rs [hunk 2]
```diff
@@ -209,15 +209,25 @@ impl ThreadPool {
         self.panic_count.load(Ordering::Relaxed)
     }
 
-    /// Sets the number of threads to use as `threads`.
+    /// Sets the number of threads to use as `num_threads`.
```
Comment (dns2utf8) [inline src/lib.rs]: Maybe remove this text, so the users of the library just see the deprecation warning.

**Comment by dns2utf8** [comment]
@frewsxcv with pleasure.

**Comment by dns2utf8** [comment]
Could you increase the version number in `Cargo.toml`?

**Comment by frewsxcv** [comment]
> Could you increase the version number in Cargo.toml?

Going to do it when I release the version

**Comment by frewsxcv** [comment]
Thanks for the review! Will merge if the tests pass
